site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -625,7 +651,91 @@ components:
          description: Allows arbitrary properties as extension to the ones
                       specified by the Hono API.
 
-# Tenant schema
+
+      PagingSortingFilteringOptions:
+         type: object
+         additionalProperties: false
+         properties:
+            "page":
+               $ref: '#/components/schemas/PagingOptions'
+            "filters":
+               type: array
+               description: A list of filter expressions. Objects must match all filters in order to be included in a result.
+               items:
+                  $ref: '#/components/schemas/FilterOption'
+            "sort":
+               type: array
+               description: A list of sorting options. If multiples options are provided, the results will be sorted following the order of the options in the list.
+               items:
+                  $ref: '#/components/schemas/SortingOption'
+
+      PagingOptions:
+         type: object
+         additionalProperties: false
+         description: options to page the results of bulk queries.
+         properties:
+            "limit":
+               type: integer
+               default: 30
+               minimum: 0
+               maximum: 200
+               description: The maximum number of objects to include in a response.
+            "offset":
+               type: integer
+               default: 0
+               minimum: 0
+               description: |
+                  The offset to use as a starting point to return results. This allows to go through the query results in
+                  a paged manner when the query returns more results than the limit.
+
+      FilterOption:
+            type: object
+            additionalProperties: false
+            description: Filters to be applied to bulk queries.
+            required:
+               - field
+               - value
+               - op
+            properties:
+               "field":
+                  type: string
+                  description: The JSON Path representation of the field to use for filtering.
```

Comment:
FMPOV we need to specify explicitly which parts of JSONPath we support here. In particular we seem to only use the *dot* notation and omit the `$` indicating the root element, right? We do seem to support array indexes but do not support predicates. Instead we use a separate property for predicates (*op*), right? Without a detailed specification of what we support here, this feature will be very hard to implement consistently ...